flexport/latitude
# Update DocumentTabs component visual styles

## Description:
Update DocumentTab to reflect new design in https://www.figma.com/file/exXS6SS14Qbfba1wyyNyqSlE/Components?node-id=253%3A11209.

## Sandbox server:
https://core-21.sandbox2.eng-dev.flexport.com/

## Jira:
https://flexport.atlassian.net/browse/LDS-154
https://flexport.atlassian.net/browse/LDS-399
https://flexport.atlassian.net/browse/LDS-395

## Manual Testing Plan:
1. Run Storybook.
2. View DocumentTab and test functionality.
3. Run tests, ensure `DocumentTab` test pass and snapshot tests having to do with `latitude.black` and `IconButton` padding pass.
4. Verify `DocumentTabs` show up correctly in Core. (https://core-21.sandbox2.eng-dev.flexport.com/carrier_bookings)
   * Filter on Client that has bookings (e.g. '10414590 Canada')
   * Select a booking and add a shipping order.
   * Observe new `DocumentTabs`.

## Screenshots:
Before:
![document_tabs_old](https://user-images.githubusercontent.com/7979583/67417643-19db5600-f57e-11e9-89e7-ffd62a9462bc.gif)

After:
![document_tabs_new](https://user-images.githubusercontent.com/7979583/67417655-1d6edd00-f57e-11e9-9594-c65199b6c98c.gif)

Before:
<img width="642" alt="Screen Shot 2019-10-23 at 11 30 21 AM" src="https://user-images.githubusercontent.com/7979583/67423118-88251600-f588-11e9-8fc6-eaa65c1062a4.png">

After:
<img width="1283" alt="Screen Shot 2019-10-23 at 11 12 38 AM" src="https://user-images.githubusercontent.com/7979583/67421858-2d8aba80-f586-11e9-8075-d8dab5c9080d.png">
